doc/introduction.md

```diff
@@ -0,0 +1,130 @@
+# Introduction
+
+This document provides an introduction of key concepts, details how
+the system boots, including failure modes, and provides links to
+other documents for further study.
+
+## CLI
+
+The command line interface (CLI, see-ell-i) is the traditional way of
+interacting with single network equipment like switches and routers.
+Today users have come to expect more advanced graphical GUIs, like a web
+interface, to manage a device or NETCONF-based tools that allow for
+managing entire fleets of installed equipment.
+
+Nevertheless, when it comes to initial deployment and debugging, it
+is very useful to know how to navigate and use the CLI.
+
+> Proceed to the [CLI User Guide](cli/tutorial.md).
+
+
+## Key Concepts
+
+The two modes in the CLI are the admin-exec and the configure context.
+When logging in to the system, from the console port or SSH, you land in
+admin-exec.  Here you can inspect the system status and do operations to
```

Comment:
Maybe we should mention that by default, you end up in `bash`, and have to run `cli` to enter admin-exec?